Examples/Statistics/WeightedSampleStatistics.cxx

```diff
@@ -67,9 +67,10 @@ class ExampleWeightFunction
   using ConstPointer = itk::SmartPointer<const Self>;
 
   /** Standard macros. */
+  /** @ITKStartGrouping */
```

Comment:
While this is a class definition, I am not sure whether we want doxygen grouping here, as this is example code which will not make it into the documentation.

Reply:
See before, though here it might be possible to do it before the integration of the PR when it is decided that no grouping should place in examples.